CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# How to contribute
```

Comment:
Suggestion: maybe include a friendly sentence thanking the person for their interest in contributing and/or mentioning that we appreciate contributions (assuming that we do :) )